setup.py

```diff
@@ -41,7 +41,7 @@ def package_data(pkg, roots):
         'markdown_xblock',
     ],
     install_requires=[
-        'XBlock<=1.9',
+        'XBlock>=2.0',
```

Comment:
I'm still wrapping my head around this, so apologies if these questions are misguided:

* Why is the PR description talking about XBlock 2 in the context of Redwood, specifically? https://github.com/openedx/edx-platform/blob/open-release/redwood.master/requirements/edx/base.txt#L1222 seems to indicate that Redwood requires XBlock 4.0.1.
* Given that the current XBlock release is 5.0.0, shouldn't this `install_requires` list say something like `XBlock<5` in order to be accurate for Redwood?

Reply:
Ah, good point. Yes the `install_requires` should be `XBlock<5`, and the commit message should say add support for XBlock 2 and higher. The breaking changes (deprecation of id_generator) were introduced in 2.0 that's why that upgrade is more significant to mention. `Quince` was running with `XBlock<2` https://github.com/openedx/edx-platform/blob/open-release/quince.master/requirements/edx/base.txt#L1212